cosipy/response/PointSourceResponse.py

```diff
@@ -74,19 +74,27 @@ def get_expectation(self, spectrum):
             spectrum_unit = spectrum.k.unit
         elif isinstance(spectrum, Line) or isinstance(spectrum, Quadratic) or isinstance(spectrum, Cubic) or isinstance(spectrum, Quartic):
             spectrum_unit = spectrum.a.unit
-        elif isinstance(spectrum, Powerlaw_Eflux) or isinstance(spectrum, Log_normal):
+        elif isinstance(spectrum, Powerlaw_flux) or isinstance(spectrum, Powerlaw_Eflux) or isinstance(spectrum, Log_normal) or isinstance(spectrum, Band_Calderone)\
```

Comment:
It seems that you are reverting @lommler's changes here, or is it something else?
https://github.com/cositools/cosipy/pull/28/files

Was that on purpose? If so, did you find an issue with the previous version? Was it needed to clean up the SpectralFitting exampple?

Reply:
Ok, thanks. I opened issue #82 to address later.

Reply:
Fixed this!